### PR TITLE
doc:update WaaG GSG based on Ubuntu.

### PR DESCRIPTION
**Pull request by shenfang2019**
Signed-off-by: fangfang.shen <fangfang.shen@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/using_windows_as_uos.rst [hunk 1]
```diff
@@ -7,105 +7,32 @@ Launch Windows as the Guest VM on ACRN
...
 
-All patches to support WaaG have been upstreamed; download them from the
-``acrn-hypervisor`` repository.
-
-Build ACRN EFI Images
-=====================
-
-#. Follow the steps described at :ref:`getting-started-building` to set up the build environment.
-#. Use the ``make`` command to compile ``acrn.efi`` and ``acrn-dm``::
+To create windows image at first, then you can launch it as Guest VM based on the Service VM.
```
Comment (dbkinder) [inline doc/tutorials/using_windows_as_uos.rst:21]: Change to:
```
In the following steps, you'll first create a Windows image
in the Service VM, and then launch that image as a Guest VM.
```

**Comment by dbkinder** [comment]
@gvancuts can you review this for technical accuracy, or can @ttzeng help?

File: doc/tutorials/using_windows_as_uos.rst [hunk 2]
```diff
@@ -7,105 +7,32 @@ Launch Windows as the Guest VM on ACRN
...
-
-   "NUC7i7DNHE", "DNKBLi7v.86A.0052.2018.0808.1344", "`link <https://downloadcenter.intel.com/download/28886?v=t>`__"
-   "NUC7i5DNHE", "DNKBLi5v.86A.0060.2018.1220.1536", "`link <https://downloadcenter.intel.com/download/28885?v=t>`__"
 
 ACRN Service VM Setup
 *********************
 
-If necessary, refer to the steps in :ref:`kbl-nuc-sdc` to set up ACRN on the
-KBL NUC. Once complete, you should be able to launch the Service VM
+Please refer to the steps in :ref:`guide <rt_industry_ubuntu_setup>` to set up ACRN
```
[the remark this answers is not in the document]
Comment (shenfang2019) [reply doc/tutorials/using_windows_as_uos.rst]: Done.

File: doc/tutorials/using_windows_as_uos.rst (hunk 1 above)
Comment (shenfang2019) [reply doc/tutorials/using_windows_as_uos.rst]: Done.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by dbkinder** [comment]
@NanlinXie why is this "LOC_Test" being run for a doc-only change, and why is it failing?

File: doc/tutorials/using_windows_as_uos.rst [hunk 3]
```diff
@@ -144,7 +72,6 @@ Create a raw disk
 
 Run these commands on the Service VM::
 
-   # swupd bundle-add kvm-host
    # mkdir /root/img
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:75]: You need to install `qemu-utils` to have the `qemu-img` available on your system. Can you add this?

Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:75]: I would actually much prefer we do **not** do this as root. Can this be done by a normal user? The `acrn` user is created as part of the Getting Started Guide and I'd recommend we use it instead

Comment (shenfang2019) [reply doc/tutorials/using_windows_as_uos.rst]: add "sudo apt-get install qemu".

Comment (shenfang2019) [reply doc/tutorials/using_windows_as_uos.rst]: change to home.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/using_windows_as_uos.rst [hunk 4]
```diff
@@ -144,9 +72,9 @@ Create a raw disk
 
 Run these commands on the Service VM::
 
-   # swupd bundle-add kvm-host
-   # mkdir /root/img
-   # cd /root/img
+   # sudo apt-get install qemu
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:75]: I don't have my Ubuntu system at hand, can you confirm that `qemu` installs `qemu-img`? I know it's part of the `qemu-utils` package but no 100% it is installed by the `qemu` package. Thanks

Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:75]: Please change the prompts from `#` to `$` (on all lines)

Comment (shenfang2019) [reply doc/tutorials/using_windows_as_uos.rst:75]: while qemu is installed, the following additional packages will be installed:
  qemu-block-extra qemu-system qemu-system-common qemu-user qemu-user-binfmt qemu-utils

Comment (shenfang2019) [reply doc/tutorials/using_windows_as_uos.rst]: Done.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/tutorials/using_windows_as_uos.rst (hunk 4 above)
Comment (gvancuts) [reply doc/tutorials/using_windows_as_uos.rst]: Unless you need any of those for other reasons, let's just install `qemu-utils`.

File: doc/tutorials/using_windows_as_uos.rst [hunk 5]
```diff
@@ -144,17 +72,17 @@ Create a raw disk
 
 Run these commands on the Service VM::
 
-   # swupd bundle-add kvm-host
-   # mkdir /root/img
-   # cd /root/img
-   # qemu-img create -f raw win10-ltsc.img 30G
+   $ sudo apt-get install qemu
```
Comment (gvancuts) [inline doc/tutorials/using_windows_as_uos.rst:75]:
```suggestion
   $ sudo apt-get install qemu-utils
```

Comment (shenfang2019) [reply doc/tutorials/using_windows_as_uos.rst]: Done.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**